package.json

```diff
@@ -108,7 +108,6 @@
 		"@hashicorp/react-alert-banner": "^7.0.1",
 		"@hashicorp/react-button": "^6.2.1",
 		"@hashicorp/react-call-to-action": "^5.1.0",
-		"@hashicorp/react-checkbox-input": "^5.0.2",
```

Comment:
As long as you've confirmed these packages aren't used across the entirely of dev portal, this LGTM!

Reply:
@nandereck Yep, this seems accurate, given we're also removing the `Sentinel` footer use of `NewsletterSignupForm` here 💯 This looks right to me!

Reply:
@zchsh - thanks for taking a look, much appreciated!